nvm.sh

```diff
@@ -54,7 +54,7 @@ nvm_command_info() {
   local INFO
   COMMAND="${1}"
   if type "${COMMAND}" | command grep -q hashed; then
-    INFO="$(type "${COMMAND}" | command sed -E 's/\(|)//g' | command awk '{print $4}')"
+    INFO="$(type "${COMMAND}" | command sed -E 's/\(|\)//g' | command awk '{print $4}')"
```

Comment:
Can you elaborate on this one? it doesn't seem to error out the `sed` call when I try it locally.